src/helpers/plainAddPlaceholder/index.js

```diff
@@ -30,6 +30,10 @@ const isContainBufferRootWithAcroform = (pdf) => {
 const plainAddPlaceholder = ({
     pdfBuffer,
     reason,
+    contact = 'emailfromp1289@gmail.com',
```

Comment:
Those don't need defaults. If they are required by the underlying code, they should be required here. If they are not required inside, they can be optional (undefined) here. Does not apply for signatureName. It needs is default value.